### PR TITLE
Changes in the way of how the package and his files are loaded

### PR DESCRIPTION
**Pull request by PabloGGaray**
Simplified form to load the package and his files, related to #17.  Included modifications in `test_chesweet.py` and `README.md` to reflect these changes.

### CODE REVIEW

File: chesweet/test/test_chesweet.py [hunk 1]
```diff
@@ -262,11 +238,10 @@ def test_compute_tors_full():
             cs2 = ef_corr-chem_list[i][1]
             x = tors_list[i]
             y = disaccharides_full.compute_tors('{}'.format(disaccharide),
-                                                cs1, cs2, ef_corr=183.4, eps=0.5)
-            if y.size != 0:# values into the distribution
+                                                cs1, cs2, ef_corr=ef_corr, eps=0.5)
+            if y.size:
```
Comment (aloctavodia) [inline chesweet/test/test_chesweet.py]: Could you change this to `if y:`?

Comment (PabloGGaray) [reply chesweet/test/test_chesweet.py]: Ok!

File: chesweet/test/test_chesweet.py [hunk 2]
```diff
@@ -277,10 +252,7 @@ def test_compute_tors_red():
             cs2 = ef_corr-chem_list[i][1]
             x = tors_list[i]
             y = disaccharides_red.compute_tors('{}'.format(disaccharide),
-                                               cs1, cs2, ef_corr=183.4, eps=0.5)
-            if y.size != 0:# values into the distribution
+                                               cs1, cs2, ef_corr=ef_corr, eps=0.5)
+            if y.size:
```
Comment (aloctavodia) [inline chesweet/test/test_chesweet.py]: Also change this to `if y:`

**Comment by aloctavodia** [comment]
I guess the change I proposed was not a good idea... checking the size was the correct choice.

**Pull request merged**